pkg/proxy/ipvs/proxier_test.go

```diff
@@ -4188,10 +4188,11 @@ func TestCleanLegacyService6(t *testing.T) {
 	ipset := ipsettest.NewFake(testIPSetVersion)
 	excludeCIDRs, _ := netutils.ParseCIDRs([]string{"3000::/64", "4000::/64"})
 	fp := NewFakeProxier(ipt, ipvs, ipset, nil, excludeCIDRs, v1.IPv4Protocol)
+	fp.ipFamily = v1.IPv6Protocol
```

Comment:
Shouldn't you just pass `v1.IPv6Protocol` to `NewFakeProxier`? It seems like you're initializing it inconsistently right now... (and in git master...)

Reply:
fixed